R/utils.R

```diff
@@ -123,13 +125,35 @@ format_se_for_publication <- function(df, selected_period=""){
   # df_filtered["margin_of_error"] = df_filtered$std_error*1.96
 }
 
-#' Format file name by appending run_id
-#' 
+#' Format file name by appending run_id and adding s3 prefix if needed
+#'
+#' @param config configuration list
+#' @param path original file path
 #' @param file_name original file name
-#' @param run_id unique identifier for the run
+#' @param platform storage platform, either "s3" or "network"
 #' @return formatted file name
 #' @export
-format_file_name <- function(file_name, run_id) {
-  file_name <- paste0(file_name, "_", run_id, ".csv")
-  return(file_name)
+format_file_name <- function(config, path, file_name, platform) {
```

Comment:
I think this should work to force the platform name to be either s3 or network, like we do below in format_path

```suggestion
format_file_name <- function(config, path, file_name, platform = c("s3", "network")) {
```